src/Core/Enums/OrganizationUserStatusType.cs

```diff
@@ -1,9 +1,10 @@
 ﻿namespace Bit.Core.Enums
 {
-    public enum OrganizationUserStatusType : byte
+    public enum OrganizationUserStatusType : short
     {
         Invited = 0,
         Accepted = 1,
-        Confirmed = 2
+        Confirmed = 2,
+        Disabled = -1,
```

Comment:
The reason I put it at the end because `Invited` is the default value, `0` is always the default backing value for an enum and I kept the default first in the list, however in evaluation of expression trees (think Lambdas/linq) you could potentially get the default from the top of the list in code-order, which could potentially create an unexpected behavior. I would prefer `Disabled` is always at the bottom of the list.